sql/migrations/20221027213705_update_coverage_field_precision.up.sql

```diff
@@ -0,0 +1,6 @@
+BEGIN;
+
+ALTER TABLE test
+  ALTER COLUMN coverage TYPE NUMERIC(5,2);
```

Comment:
same comment as prior PR to remove this because coverage is intended to be between `0 <-> 1` (just a reminder)